contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -4001,6 +4001,84 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql IMMUTABLE STRICT PARALLEL SAFE;
 
+-- wrapper functions for concat --
+CREATE OR REPLACE FUNCTION sys.concat(VARIADIC args sys.VARCHAR[] DEFAULT '{}')
```

Comment:
Since without `default` this function would consider atleast one argument to hit this definition, so case like `CONCAT()` will give error `CONCAT function requires 1 argument` without default which would be misleading. So, marking the argument as `DEFAULT` will make it hit the definition and throw a more relevant error: `The concat function requires 2 to 100 arguments.`